explorer/fuzzing/ast_to_proto.cpp

```diff
@@ -10,7 +10,7 @@
 #include "explorer/ast/expression.h"
 #include "llvm/Support/Casting.h"
 
-namespace Carbon {
+namespace Carbon::Testing {
```

Comment:
The changes in this file, and a few other files with similar changes, seem unrelated to the topic of this PR. Are these the changes to avoid merge conflicts you alluded to?